purple-pill/templates/menu.html

```diff
@@ -1,18 +1,21 @@
-<div id="project">
-    <localmenu class="localmenu" class="left">
-        <div class="lm-head" ng-mousedown="data.selectMenu('project')">
-            <b>Project</b> <span id="project-name" ng-model="data.projectName" class="project-rename"></span>
-        </div>
-        <div class="lm-body" ng-if="data.localmenu.project">
-            <div class="lm-button" ng-mouseup="newDocument()">New</div>
-            <div class="lm-divider"></div>
-            <div class="lm-button" ng-mouseup="closeDocument()">Close</div>
-            <div class="lm-divider"></div>
-            <div class="lm-button" ng-mouseup="renameProject()">Rename project</div>
-        </div>
-    </localmenu>
+<div id="project" class="left">
+    <b>Project</b> <span id="project-name" ng-model="data.projectName" class="project-rename"></span>
 </div>
-<!-- end of project -->
+<!--<div id="project">-->
+    <!--<localmenu class="localmenu" class="left">-->
+        <!--<div class="lm-head" ng-mousedown="data.selectMenu('project')">-->
+            <!--<b>Project</b> <span id="project-name" ng-model="data.projectName" class="project-rename"></span>-->
+        <!--</div>-->
+        <!--<div class="lm-body" ng-if="data.localmenu.project">-->
+            <!--<div class="lm-button" ng-mouseup="newDocument()">New</div>-->
+            <!--<div class="lm-divider"></div>-->
+            <!--<div class="lm-button" ng-mouseup="closeDocument()">Close</div>-->
+            <!--<div class="lm-divider"></div>-->
+            <!--<div class="lm-button" ng-mouseup="renameProject()">Rename Project</div>-->
```

Comment:
renameProject worked (before). Please leave it  in and I will check out whats wrong with it.